internal/apis/meta/v1/conversion.go

```diff
@@ -20,7 +20,7 @@ import (
 	"k8s.io/apimachinery/pkg/conversion"
 
 	"github.com/jetstack/cert-manager/internal/apis/meta"
-	"github.com/jetstack/cert-manager/pkg/apis/meta/v1"
+	v1 "github.com/jetstack/cert-manager/pkg/apis/meta/v1"
```

Comment:
Nit: this import is named `cmmeta` in most/all other places in the code base i.e https://github.com/jetstack/cert-manager/blob/master/pkg/controller/certificates/requestmanager/requestmanager_controller.go#L43 perhaps we want to keep/enforce that as a standard?

Reply:
I've also renamed k8s networking v1 to netv1